xml/System.Windows.Forms/MdiClient.xml

```diff
@@ -35,7 +35,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- You should not create an <xref:System.Windows.Forms.MdiClient> control. A form creates and uses the <xref:System.Windows.Forms.MdiClient> when you set the <xref:System.Windows.Forms.Form.IsMdiContainer%2A> property to `true`.  
+ Don't create an <xref:System.Windows.Forms.MdiClient> control. A form creates and uses the <xref:System.Windows.Forms.MdiClient> when you set the <xref:System.Windows.Forms.Form.IsMdiContainer%2A> property to `true`.  
```

Comment:
```suggestion
 Don't create an <xref:System.Windows.Forms.MdiClient> control. A form creates and uses the <xref:System.Windows.Forms.MdiClient> when you set the <xref:System.Windows.Forms.Form.IsMdiContainer%2A> property to <see langword="true"/>.  
```

Reply:
This part is correct. This is wrapped in a CDATA block, so we can use Markdown here. Remarks don't go into IntelliSense.